python-lib/recipe_config_loading.py

```diff
@@ -3,6 +3,11 @@
 logger = SafeLogger("Time series preparation plugin")
 
 
+def check_input_parameters(recipe_config, dataset_columns):
```

Comment:
maybe you should rename it check_time_column_parameter as it only does that

Reply:
renamed in https://github.com/dataiku/dss-plugin-timeseries-preparation/pull/23/commits/53fadcf186122fa512b8fea1b94b946651334f35